docs/running_rune_with_occlum.md

```diff
@@ -128,7 +142,7 @@ export OCCLUM_INSTANCE_DIR=occlum-app
 yum install -y libseccomp
 docker run -it --rm --runtime=rune \
   -e ENCLAVE_TYPE=intelSgx \
-  -e ENCLAVE_RUNTIME_PATH=/usr/lib/liberpal-occlum.so \
+  -e ENCLAVE_RUNTIME_PATH=/opt/occlum/build/lib/libocclum-pal.so.0.14.0 \
```

Comment:
I assume this does right because occlum pal package will install pal to /opt/occlum/build/lib/libocclum-pal.so.0.14.0 right?

Reply:
If so, why install pal to such a path? Why nit /usr/lib/liberpal-occlum.so?